extension/package.json

```diff
@@ -59,15 +30,75 @@
       "last 1 firefox version"
     ]
   },
-  "devDependencies": {
-    "@stellar/eslint-config": "^1.0.3",
+  "babel": {
+    "presets": [
+      "@babel/preset-typescript",
+      "@babel/preset-react",
+      [
+        "@babel/preset-env",
+        {
+          "targets": {
+            "node": "current"
+          }
+        }
+      ]
+    ]
+  },
+  "jest": {
+    "collectCoverageFrom": [
+      "src/**/*.{js,jsx,mjs}"
+    ],
+    "setupFiles": [
+      "<rootDir>/config/jest/setupTests.ts"
+    ],
+    "setupFilesAfterEnv": [
+      "<rootDir>/config/jest/extendJest.ts"
+    ],
+    "testEnvironment": "node",
+    "testURL": "http://localhost",
+    "transform": {
+      "^.+\\.(js|jsx|ts|tsx|mjs)$": [
+        "babel-jest"
+      ]
+    },
+    "moduleNameMapper": {
+      "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/__mocks__/fileMock.js",
+      "\\.(css|less)$": "<rootDir>/__mocks__/styleMock.js"
+    },
+    "moduleFileExtensions": [
+      "js",
+      "jsx",
+      "json",
+      "node",
+      "mjs",
+      "ts",
+      "tsx"
+    ],
+    "moduleDirectories": [
+      "node_modules",
+      "<rootDir>/src",
+      "<rootDir>/.."
+    ]
+  },
+  "dependencies": {
```

Comment:
I merged `devDependencies` and `dependencies`. Since this is an app there isn't really a situation where the division matters